include/openssl/lhash.h.in

```diff
@@ -207,7 +202,7 @@ void OPENSSL_LH_node_usage_stats_bio(const OPENSSL_LHASH *lh, BIO *out);
 #  define DEFINE_LHASH_OF_DEPRECATED(type)
 # endif
 
-# define DEFINE_LHASH_OF(type) \
+# define DEFINE_LHASH_OF_2(type) \
```

Comment:
IMO it should be named DEFINE_LHASH_OF_EX

Reply:
Yes - I agree with @t8m here